Wikipedia/Code/WKWebView+LoadAssetsHtml.m

```diff
@@ -25,8 +25,31 @@ - (void)loadHTML:(NSString*)string withAssetsFile:(NSString*)fileName scrolledTo
     }
     
     WMFProxyServer *proxyServer = [WMFProxyServer sharedProxyServer];
+    
+    
+    NSUInteger windowPixelWidth = (NSUInteger)(MIN(self.bounds.size.width, self.bounds.size.height) * self.window.screen.scale);
+    
+    NSUInteger targetImageWidth = 320;
+    
+    if (windowPixelWidth < 640) {
+        targetImageWidth = 320;
+    } else if (windowPixelWidth < 800) {
+        targetImageWidth = 640;
+    } else if (windowPixelWidth < 1024) {
+        targetImageWidth = 800;
```

Comment:
I think 640 should be the cutoff. 

This image from the Obama article has a canonical size of 4800 x 6000, so it's a good example for an image that has lots of headroom for requesting higher resolutions...

On iPad (open these images in separate tabs to compare):

- **Before this PR**, this image was fetched at 340px width, and the entire page transfer size was **2.54 MB**:
![pre change](https://cloud.githubusercontent.com/assets/3143487/16324031/d9df39c8-3964-11e6-972c-645237139a73.png)

- **This PR fetched at 640px width**, and entire page transfer size of **3.36 MB**:
![this pr at 640](https://cloud.githubusercontent.com/assets/3143487/16324089/82c23e28-3965-11e6-912e-6d6e3f09a611.png)

- **This PR fetched at 800px width**, and entire page transfer size of **4.10 MB**:
![this pr at 800](https://cloud.githubusercontent.com/assets/3143487/16324130/074b5922-3966-11e6-8a47-b9039cf18f1b.png)

640px looks great IMO and it's more detailed than the 340px we were formerly fetching.

iPhone 6 transfer sizes for Obama article:
 320  1.70 MB
 340  2.13 MB < Before this PR
 640  3.36 MB < After this PR
 800  4.10 MB
1024  6.48 MB
1280  7.77 MB
1920  11.7 MB
2560  18.9 MB
2880  20.7 MB